Games/Retro_ML.SuperMarioBros/Neural/Scoring/CoinsScoreFactor.cs

```diff
@@ -79,7 +68,7 @@ public void LevelDone()
 
         public IScoreFactor Clone()
         {
-            return new CoinsScoreFactor() { IsDisabled = IsDisabled, ScoreMultiplier = ScoreMultiplier };
+            return new CoinsScoreFactor() { IsDisabled = IsDisabled, ScoreMultiplier = ScoreMultiplier, ExtraFields = ExtraFields };
```

Comment:
```suggestion
            return new CoinsScoreFactor() { IsDisabled = IsDisabled, ScoreMultiplier = ScoreMultiplier };
```

ExtraFields will be removed soon, no need to add them to Clone()